index.html

```diff
@@ -0,0 +1,132 @@
+<!doctype html>
+<html>
+  <head>
+     <meta name="viewport" content="width=device-width,initial-scale=1">
+    <link href="scss/styles.css" type="text/css" rel="stylesheet"/>
+  </head>
+  <body>
+    <header class="header-wrapper wrapper">
+        <img class="main-logo" src="img/minimo_logo.png" alt="minimo_logo" />
+        <nav class="main-nav hamburger-wrapper">
+          <svg class="hamburger" height="32px" id="Layer_1" style="enable-background:new 0 0 32 32;" version="1.1" viewBox="0 0 32 32" width="32px" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path d="M4,10h24c1.104,0,2-0.896,2-2s-0.896-2-2-2H4C2.896,6,2,6.896,2,8S2.896,10,4,10z M28,14H4c-1.104,0-2,0.896-2,2  s0.896,2,2,2h24c1.104,0,2-0.896,2-2S29.104,14,28,14z M28,22H4c-1.104,0-2,0.896-2,2s0.896,2,2,2h24c1.104,0,2-0.896,2-2  S29.104,22,28,22z"/></svg>
+          <ul class="main-menu dropdown">
+            <li class="main-menu-item"><a href="#">lifestyle</a></li>
+            <li class="main-menu-item"><a href="#">photodiary</a></li>
+            <li class="main-menu-item"><a href="#">music</a></li>
+            <li class="main-menu-item"><a href="#">travel</a></li>
+          </ul>
+         </nav>
+    </header>
+    <main class="main-content-wrapper">
+      <section id="main-article">
+        <div class="wrapper">
+          <img src="img/pics/main_post_img.jpg" alt="Main Article Image" class="main-image"/>
+          <article class="main-article">
+            <span class="category">photodiary</span>
+            <h2 class="main-article-title">The Perfect weekend getaway</h2>
```

Comment:
tutaj stosujesz coś a la BEM jeśli chodzi o nazewnictwo klas. Gdyby miało być jak w `BEM` to powinno być `main-article__tile` ;)

Reply:
wrzucielm sobie BEM do poczytania